share/spice/metar/metar.js

```diff
@@ -0,0 +1,29 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_metar = function(api_result) {
+        
+        if (!api_result || api_result.Error) {
+            return Spice.failed('metar');
+        }    
+
+        Spice.add({
+            id: "metar",
+            name: 'Weather',
+            data: api_result,
+            meta: {
+                sourceName: "Aviation Weather REST API",
+                sourceUrl: "http://avwx.rest"
+            },
+            normalize: function(item) {
+                return {
+                    title: item["Station"], 
+                    subtitle: item["Time"],
```

Comment:
I just spoke to a friend that's a pilot and he mentioned TAF data would be great to include. I see the API provides it: http://avwx.rest/api/taf.php?station=KLAX&options=summary&format=JSON

Do you have any ideas how we could incorporate that data as well?

Would it help to use our `List` template and display a table with rows of info?